app/views/runs/_title.slim

```diff
@@ -1,72 +1,64 @@
-h3
-  = run.to_s
-- if run.completed?(Run::REAL) && run.completed?(Run::GAME)
-  = render partial: 'runs/timing_switch', locals: {timing: timing}
-h5.mt-3
-  div = format_milliseconds(run.duration_ms(timing))
-  - if run.user.try(:name)
-    small
-      span by #{link_to run.user, user_path(run.user), class: 'stealth-link user-link'}
-      - if run.user.silver_patron?
-        .gold-split title='Patron'
-h6
+h1 = run.to_s
+h5
+  span> by
+  span = user_badge(run.user)
+  .badge
+    i.fas.fa-check>
+    span Personal Best
```

Comment:
Every single run is now a personal best, including unowned runs.

Reply:
lol